lib/compress/huf_compress.c

```diff
@@ -311,29 +311,34 @@ static U32 HUF_setMaxHeight(nodeElt* huffNode, U32 lastNonNull, U32 maxNbBits)
     return maxNbBits;
 }
 
-
 typedef struct {
     U32 base;
     U32 current;
 } rankPos;
 
-static void HUF_sort(nodeElt* huffNode, const unsigned* count, U32 maxSymbolValue)
+typedef nodeElt huffNodeTable[HUF_CTABLE_WORKSPACE_SIZE_U32];
+
+typedef struct {
+  rankPos rankPosition[32];
+  huffNodeTable huffNodeTbl;
```

Comment:
minor : it feels proper to reverse positions of `huffNodeTbl` and `rankPosition` in the `struct`.
That's because `huffNodeTbl`'s lifetime is the larger one (it starts before and ends after `rankPosition`).